packages/components/psammead-topic-tags/src/index.jsx

```diff
@@ -54,8 +55,13 @@ const SingleTopicTagItem = styled.div`
   }
 
   a > span {
```

Comment:
On the call we discussed reducing this to one element – `a`  and pruning the `span`, which was probably used in a previous iteration.